app/helpers/application_helper.rb

```diff
@@ -116,6 +116,23 @@ def deeds_for(options={})
     render({ :partial => 'deed/deeds', :locals => { :limit => limit, :deeds => deeds, :options => options} })
   end
 
+  def show_prerender(prerender, locale) 
+    begin
+      prerenders = JSON.parse(prerender)
+      if prerenders.key? locale.to_s
+        # show prerender in specified locale
+        prerenders[locale.to_s]
+      else
+        # prerender doesn't have specified locale, show first fallback that prerender has
+        fallback = I18n.fallbacks[locale].select { |locale| prerenders.key? locale.to_s }.first
```

Comment:
I'm pretty sure this could be made a bit more comprehensible using something like `I18n.fallbacks[locale] & prerenders.keys` to grab the fallback.

Reply:
Thanks that makes more sense